src/openzaak/utils/permissions.py

```diff
@@ -46,6 +47,9 @@ def get_main_object(self, obj, permission_main_object):
         return getattr(obj, permission_main_object)
 
     def has_permission(self, request: Request, view) -> bool:
+        if not view.action:
+            raise MethodNotAllowed(request.method)
```

Comment:
This is a bit bold - currently we do use `ViewSet` almost everywhere, but this will break for bare `APIView` views.

edit: I'm changing the implementation based on DRFs view code in `rest_framework.views.APIView.dispatch`

```python
            self.initial(request, *args, **kwargs)

            # Get the appropriate handler method
            if request.method.lower() in self.http_method_names:
                handler = getattr(self, request.method.lower(),
                                  self.http_method_not_allowed)
            else:
                handler = self.http_method_not_allowed
```